carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/R.java

```diff
@@ -119,6 +119,17 @@
     public void put(String key, String value) {
         propertiesHolder.get(resourceFile).put(key, value);
     }
+    
+    /**
+     * Verify if key is declared in data map.
+     * @return boolean
+     */
+    public boolean containsKey(String key) {
+    	boolean res = CONFIG.resourceFile.equals(resourceFile) ? propertiesHolder.get(resourceFile).containsKey(key)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "res". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1488)